jdbc/src/java/org/apache/hive/jdbc/HiveDatabaseMetaData.java

```diff
@@ -1227,4 +1232,13 @@ private TGetInfoResp getServerInfo(TGetInfoType type) throws SQLException {
     Utils.verifySuccess(resp.getStatus());
     return resp;
   }
+
+  private boolean getHiveDefaultNullsLast() {
+    boolean response = ConfVars.HIVE_DEFAULT_NULLS_LAST.defaultBoolVal;
```

Comment:
HIVE_DEFAULT_NULLS_LAST could be overridden on the HiveServer2 using hive-site.xml settings. We need to get a value of this from HS2. 
I am not sure if we get the current settings from HS2 (i need to check).

Reply:
In Beeline we pick the Hive config and prefix it with hiveconf: and put it in the JDBCParameters.hiveConf. So, I have added the prefix, while looking up for the relevant key in the map. Not sure if there is any other place, where we send the hive conf differently.

Reply:
I think @thejasmn concern is valid. If user specifies the property value in the connection parameters, then we will retrieve it here. However, if it is not present, we probably need to establish a connection to HS2 to retrieve the parameter value, since it may not be necessarily the default?

Reply:
@jcamachor @thejasmn Looks like we already open the session with HS2 in HiveConnection, but when I see the OpenSession implementation in ThriftCLIService, looks like we only send one HiveConf configuration : 
 resp.setConfiguration(Collections
          .singletonMap(HiveConf.ConfVars.HIVE_SERVER2_THRIFT_RESULTSET_DEFAULT_FETCH_SIZE.varname,
              Integer.toString(fetchSize))); Should I add HIVE_DEFAULT_NULLS_LAST here or we need to expose a new method ?

Reply:
@kishendas , I think it makes sense to return that property value too when the session is open since the mechanism is already in place and adding an additional property value should not create much overhead.

Reply:
@jcamachor @thejasmn Please check if the changes look good.

Reply:
New code makes sense to me. @thejasmn could you take a quick look in case I am missing anything?